eng/Version.Details.xml

```diff
@@ -9,4 +9,10 @@
       <Sha>26d9440e6683b44c4db5f45b25e9c857a6563bb8</Sha>
     </Dependency>
   </ProductDependencies>
+  <ToolsetDependencies>
```

Comment:
What's the difference between `<ToolsetDependencies>` and `<ProductDependencies>`?

Reply:
I don't think there's any functional difference, the description from arcade says this:

> ### Toolset vs. Product Dependencies
> There are two types of dependencies: 'Product' and 'Toolset'. Choosing between them involves answering the question:
>
> "Does my repository repackage the input dependency's binaries, or information about those binaries in the outputs it creates for the product?" If so, that dependency is 'Product', otherwise, it's 'Toolset'.

https://github.com/dotnet/arcade/blob/6cc4c1e9e23d5e65e88a8a57216b3d91e9b3d8db/Documentation/Darc.md#toolset-vs-product-dependencies

Reply:
So things from dotnet/arcade or the dotnet-eng NuGet feed are definitely "toolset", that makes sense. 👍